### PR TITLE
Feature/style update

### PR DESCRIPTION
**Pull request by ZenVega**
equivalent figma file: https://www.figma.com/file/s12w00ODssXTtPQfSizlGj/TSB-Sachen?node-id=56%3A1510

### CODE REVIEW

**Review by vogelino — APPROVED** [review]
Top! 👍

File: src/components/Funfact/Funfact.module.scss [hunk 1]
```diff
@@ -12,7 +12,7 @@
   .text {
     margin-left: 10px;
     width: 100%;
-    color: $blue;
+    color: $corporate-blue;
```
Comment (ff6347) [inline src/components/Funfact/Funfact.module.scss:15]: IMO is the corporate redundant information. Could have kept it as blue

File: src/components/Headline/Headline.module.scss [hunk 2]
```diff
@@ -1,10 +1,10 @@
...
 :local {
   .headline {
-    line-height: 120%;
-    margin: 10px auto 20px auto;
-    font-size: $font-size-xxl;
-    font-weight: normal;
+    line-height: 100%;
+    margin: 20px auto 35px;
+    font-size: $font-size-xl;
+    font-weight: $bold-text;
```
Comment (ff6347) [inline src/components/Headline/Headline.module.scss:7]: text is also redundant information here

File: src/components/Icon/Icon.module.scss [hunk 3]
```diff
@@ -3,7 +3,7 @@
   .iconWrapper {
     width: 36px;
     height: 36px;
-    background-color: $dark-grey--7;
+    background-color: $dark-grey--6;
```
Comment (ff6347) [inline src/components/Icon/Icon.module.scss:6]: Having the 6 in here is pretty generic naming. Is there any better naming solution for that grey?

File: src/content.js [hunk 4]
```diff
@@ -163,7 +163,7 @@ const content = {
         img: {
           src: "assets/images/smart-city_ODIS.jpg",
           alt: "image tag",
-          copyright: "© 2022 Technologiestiftung Berlin",
+          copyright: "© CityLAB Berlin",
```
Comment (ff6347) [inline src/content.js:166]: Is this right? ODIS is not a citylab project

File: src/styles/constants.scss [hunk 5]
```diff
@@ -1,16 +1,11 @@
...
-$dark-grey--2: #666666;
-$dark-grey--3: #8C8C8C;
-$dark-grey--4: #B3B3B3;
-$dark-grey--5: #D9D9D9;
-$dark-grey--6: #ECECEC;
-$dark-grey--7: #F5F5F5;
-$dark-grey--8: #FAFAFA;
+$dark-grey--1: #3B3B3A;
+$dark-grey--2:#6F6F6E;
+$dark-grey--6: #ECECED;
```
Comment (ff6347) [inline src/styles/constants.scss:4]: what about 3 to 5?

**Review by ff6347 — COMMENTED** [review]
I might be late to the party, but here are some of my 50 cents

**Pull request merged**